js/PulsarFormComponent.js

```diff
@@ -2,9 +2,9 @@
 
 var $ = require('jquery');
 
-require('../libs/pikaday/plugins/pikaday.jquery');
-require('../libs/select2/dist/js/select2.min');
-require('../libs/spectrum/spectrum');
+require('../node_modules/pikaday/plugins/pikaday.jquery');
```

Comment:
I'm fairly certain this will do `'pikaday/plugins/pikaday.jquery'`, webpack will automatically attempt to resolve require statements to the nearest `node_modules` folder recursively (in an upwards direction, not sure if "recursively" can mean in an upwards and downwards direction)

Reply:
I wikiapedia'd it, I think recursion applies to both directions 🍰